init.py

```diff
@@ -19,4 +19,4 @@ def cliked():
 btn = Button(root, text = "Click me" ,
```

Comment:
Тут лучше вместо Click me написать привет

```suggestion
btn = Button(root, text = "привет" ,
```